content/blog/2025/2025-09-25-neue-websites.md

```diff
@@ -0,0 +1,85 @@
+
+**Die Space-Seite ist unter `mainframe.io` zu finden:**
+
+[https://www.mainframe.io/](https://www.mainframe.io/)
+
+Auf der **KtT-Seite** finden sich Infos, die sich eher an die Allgemeinheit,
+sowie öffentliche Stellen und potenzielle Förderpartner richten. Zusätzlich
+liegen dort die Vereinssatzung und Informationen zu Mitgliedschaft im Verein
+sowie Spendenmöglichkeiten.
+
+Die **Mainframe-Seite** ist Anlaufstelle für alles, was im Space Tag für Tag passiert. Hier
+finden sich auch das IFS, Blogbeiträge aus Projekten im Space sowie der
+Terminkalender.
+
+Aber das ist natürlich noch nicht alles! Wir haben Inhalte grundlegend
+überarbeitet, neue Seiten hinzugefügt und Menüstrukturen verbessert.
+
+## Was ist neu auf der Mainframe-Seite?
+
+Die Space Seite, [mainframe.io](https://mainframe.io) gibt es weiterhin. Sie
+behält bewusst das grundsätzliche Design der alten Seite bei. Bei den Inhalten
+und bei der Technik hat sich allerdings eine Menge getan:
+
+* **Neue Menüführung und About-Seiten:**
+  Die Navigation haben wir vereinfacht, sodass der Terminkalender und die
+  Blogposts schneller zu finden sind. Außerdem gibt es mehrere neue Seiten unter
+  "Über uns", die erklären, was der Space ist und wie man zu uns findet.
+* **Neue Bereichs-Seiten:**
+  Die einzelnen Bereiche im Space sowie die Maschinen und Werkzeuge dort sind
+  neu beschrieben. Dafür haben wir neue Fotos gemacht und Inhalte neu gruppiert.
+  Einige Seiten können noch mehr im Detail beschrieben werden -- das holen wir
+  in den kommenden Wochen nach. Gerne mit deiner Hilfe!
+* **KtT-Inhalte verschoben:**
+  Die Inhalte zum Verein, wie Mitgliedschaft, Satzung und Sponsoren sind jetzt
+  auf der neuen KtT-Seite zu finden.
+* **Schnellere Ladezeiten**:
+  Bilder und Seiten -- insbesondere das IFS -- laden jetzt schneller, auch auf
+  der mobilen Seite. Den Lade-Pacman wirst du hoffentlich nur noch selten sehen.
+* **Technisches Fundament erneuert:**
+  Wir setzen jetzt einen moderneren Static-Site-Generator ein, der schneller und
+  einfacher zu handhaben ist als der alte. Der neue Generator verwendet nun
+  Markdown als Template-Sprache. Dafür mussten wir zwar alle Inhalte einzeln
+  migrieren, aber das hat sich gelohnt: In Zukunft ist es deutlich einfacher,
+  eigene Beiträge oder Änderungen beizusteuern.
+* **Aufgeräumt:**
+  Einige alte Funktionen, zum Beispiel aus dem alten Kalender, haben wir
+  komplett entfernt.
+
+## Wie geht's weiter?
+
+Die Website lebt von den Inhalten dort! Egal ob du über ein Projekt berichten
+möchtest, oder einfach ein IFS-Bild teilen willst: Wir freuen uns auf deine
+Beiträge!
+
+Falls du dich mit GitHub oder dem IFS-Workflow (noch) nicht so wohl fühlst,
+sprich uns gerne direkt im Space an und wir helfen dir weiter.
+
+Also -- schau dich direkt mal um:
+
+* [mainframe.io](https://mainframe.io)
+* [kreativitaet-trifft-technik.de/](https://www.kreativitaet-trifft-technik.de/)
```

Comment:
You can remove the / here

Reply:
Fixed. Found two other places in this article and fixed them as well.